packages/hash/api/src/seed-data/index.ts

```diff
@@ -0,0 +1,87 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { OrgModel, OrgSize, UserModel } from "../model";
+import { workspaceAccountId } from "../model/util";
+import { ensureDevUsersAreSeeded } from "./dev-users";
+import { PageList, seedPages } from "./seed-pages";
+
+const seedUserContent = async (
+  user: UserModel,
+  sharedOrg: OrgModel,
+  params: {
+    graphApi: GraphApi;
+    logger: Logger;
+  },
+): Promise<void> => {
+  const { graphApi, logger } = params;
+
+  await user.joinOrg(graphApi, {
+    org: sharedOrg,
+    responsibility: "Member",
+    actorId: workspaceAccountId,
+  });
+
+  logger.info(
+    `User with shortname = "${user.getShortname()}" joined org with shortname = '${sharedOrg.getShortname()}'`,
+  );
```

Comment:
Nit: I wouldn't think of an org to be "user content". I think `joinOrg` can just be called inline with `seedUserContent`, or just call `joinOrg` and `seedPages` side by side as this `seedUserContent` method isn't reused anywhere anyway.

Reply:
That's a good point, removed it in 97c416e091289ed7ac7ed1580dd04cca58fc472f